google/cloud/sqlalchemy_spanner/sqlalchemy_spanner.py

```diff
@@ -341,6 +341,9 @@ def post_create_table(self, table):
         cols = [col.name for col in table.primary_key.columns]
         post_cmds = " PRIMARY KEY ({})".format(", ".join(cols))
 
+        if table._prefixes == ["TEMPORARY"]:
```

Comment:
nit: prefer to check if it's contained in the list in case other prefixes are used as well

```suggestion
        if "TEMPORARY" in table._prefixes:
```